### PR TITLE
Sprint 4/issue 19#create passenger screen

### PR DESCRIPTION
**Pull request by landillo**
#What we do?

We create the select passenger screen and implemented the functionality.

Captura de pantalla 2023-07-12 a la(s) 22 27 28

### CODE REVIEW

File: src/components/PassengersSelector/PassengersSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { View } from 'react-native';
+import { Picker } from 'react-native-wheel-pick';
+import React, { useState } from 'react';
+import Icon from 'react-native-vector-icons/Ionicons'
+import { styles } from './styles';
+
+
```
Comment (wiltrouble) [inline src/components/PassengersSelector/PassengersSelector.tsx]: remove this row

File: src/screens/bookingScreensStyle.ts
Comment (wiltrouble) [inline src/screens/bookingScreensStyle.ts:1]: Why is empty this file?

File: src/components/PassengersSelector/PassengersSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          name="caret-back"
+          size={30}
+          color="#6170F7"
+        />      
+      </View>
+    </View>
+  );
+};
+
+
```
Comment (wiltrouble) [inline src/components/PassengersSelector/PassengersSelector.tsx]: remove the empty rows

File: src/components/PassengersSelector/styles.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from "react-native";
+
```
Comment (wiltrouble) [inline src/components/PassengersSelector/styles.ts:2]: remove this empty row

File: src/components/PassengersSelector/PassengersSelector.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import React, { useState } from 'react';
+import Icon from 'react-native-vector-icons/Ionicons'
+import { styles } from './styles';
+
+
+
+export const PassengersSelector = ({ onSelectPassengers }: any) => {
+    const [selectedPassenger, setSelectedPassenger] = useState(1);
+
+    const onPickerSelect = (index: any) => {
```
Comment (wiltrouble) [inline src/components/PassengersSelector/PassengersSelector.tsx]: don't use generic types

**Review by wiltrouble — CHANGES_REQUESTED** [review]

**Review by montsegv-2 — APPROVED** [review]

**Review by Lizzflores — APPROVED** [review]

**Review by wiltrouble — APPROVED** [review]

**Pull request merged**